app.R

```diff
@@ -2,9 +2,17 @@
 # To deploy, run: rsconnect::deployApp()
 # Or use the blue button on top of this file
 
+localDevelopment = FALSE
+
 if (nchar(Sys.getenv("R_CONFIG_ACTIVE"))==0) {
-	Sys.setenv(R_CONFIG_ACTIVE = "default") # Replace "default" with your config
+  if(localDevelopment == TRUE) {
+    Sys.setenv(R_CONFIG_ACTIVE = "testing")
+    options(shiny.port = 8100)
+  } else {
+    Sys.setenv(R_CONFIG_ACTIVE = "default")
+  }
 }
+
 pkgload::load_all()
 options("golem.app.prod" = TRUE)
-stopadforms::run_app() # add parameters here (if any)
+stopadforms::run_app() # add parameters here (if any)
```

Comment:
do you think we need a new line at the end of this file?

Reply:
I don't think so for this an R file.   There needs to be a new line for .yml files, though.  All operating systems pass.